### PR TITLE
Bam to cram support

### PR DESCRIPTION
**Pull request by smk5g5**
Bam to CRAM support added to rnaseq star fusion cwl

### CODE REVIEW

File: definitions/pipelines/rnaseq_star_fusion.cwl [hunk 1]
```diff
@@ -192,3 +192,16 @@ steps:
             bam: mark_dup/sorted_bam
         out:
             [metrics, chart]
+    bam_to_cram:
+        run: ../tools/bam_to_cram.cwl
+        in:
+          reference: cdna_fasta
```
Comment (chrisamiller) [inline definitions/pipelines/rnaseq_star_fusion.cwl:198]: The cdna fasta is not the same as the reference sequence fasta.  It contains one fasta entry for each transcript.  The reference fasta is needed here, either derived from the star directory, or passed in explicitly

Comment (smk5g5) [reply definitions/pipelines/rnaseq_star_fusion.cwl:198]: Mybad!

**Review by tmooney — APPROVED** [review]
+1 🦀

**Review by chrisamiller — APPROVED** [review]
+1  looks good!

**Pull request merged**